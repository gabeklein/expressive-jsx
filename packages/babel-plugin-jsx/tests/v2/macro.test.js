transform("multi-argument style", `
  () => {
    value: foo, bar, baz;

    <this>Hello World</this>
  };
`);

transform("output attributes", `
  () => {
    bg: red;
    
    <this />
  };
`);

transform("output style of same name", `
  () => {
    marginH: 20;

    <this>Hello World</this>
  };
`)