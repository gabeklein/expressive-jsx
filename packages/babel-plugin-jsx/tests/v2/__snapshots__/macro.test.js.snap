// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`multi-argument style 1`] = `
  () => <div className="element_41k">Hello World</div>;
  css(\`
  	.element_41k {
  		value: foo bar baz;
  	}
  \`);
`;

exports[`output attributes 1`] = `
  () => <div className="element_41k" />;
  css(\`
  	.element_41k {
  		background: red;
  	}
  \`);
`;

exports[`output style of same name 1`] = `
  () => <div className="element_41k">Hello World</div>;
  css(\`
  	.element_41k {
  		margin-left: 20px;
  		margin-right: 20px;
  	}
  \`);
`;
